tests/testthat/test-expr_datetime.R

```diff
@@ -18,6 +18,18 @@ test_that("pl$lit posix", {
     pl$lit(as.POSIXct("2022-01-01", tz = "GMT"))$to_r(),
     as.POSIXct("2022-01-01", tz = "GMT")
   )
+
+
+  x = as.POSIXct(
+    c(
+      "2020-01-01 13:45:48.343", "2020-01-01 13:45:48.815"
+      , "2020-01-01 13:45:49.289", "2020-01-01 13:45:49.974"
+      , "2020-01-01 13:45:51.190", "2020-01-01 13:45:51.631"
+    ), tz = "UTC"
+  )
+  expect_identical(pl$lit(x)$to_r(), x)  #preserve millisecond precision
+  expect_failure(expect_identical(pl$lit(x)$to_r(), x+0.001)) #control, detect 1ms offset
```

Comment:
I feel this test should not be here. This is not a test for expr, but for type conversion.

However, we did not know where such a test is currently put together.